sdk/resourcemanager/azure-resourcemanager-appservice/src/test/java/com/azure/resourcemanager/appservice/OneDeployTests.java

```diff
@@ -84,9 +85,9 @@ public void canPushDeployJar() throws Exception {
                 .create();
 
         // deploy
-        File jarFile = new File("gateway.jar");
+        File jarFile = new File("helloworld.jar");
```

Comment:
Built from [Azure Samples](https://github.com/Azure-Samples/java-docs-spring-hello-world).

gateway.jar reads config from http://config:8888, resulting in `UnknownHostException`.